tests/examples/opinions/united_states/uscgcca_example.compare.json

```diff
@@ -0,0 +1,167 @@
+    "case_names": "ORDER - In re Tucker Petition 2019 WL 5280525",
+    "download_urls": "https://media.defense.gov/2019/Oct/23/2002199220/-1/-1/0/IN%20RE%20ETHAN%20W.%20TUCKER%20-%20ORDER.PDF",
+    "precedential_statuses": "PUBLISHED",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "neutral_citations": null,
+    "west_citations": "2019 WL 5280525",
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2019-10-04",
+    "case_names": "UNITED STATES V MARTINEZROQUE",
+    "download_urls": "https://media.defense.gov/2019/Oct/07/2002191316/-1/-1/0/UNITED%20STATES%20V%20MARTINEZROQUE%20(MERITS).PDF",
+    "precedential_statuses": "PUBLISHED",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "neutral_citations": null,
+    "west_citations": null,
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2019-09-25",
+    "case_names": "UNITED STATES v. OLSEN",
+    "download_urls": "https://media.defense.gov/2019/Sep/25/2002186965/-1/-1/0/UNITED%20STATES%20V.%20OLSEN%20__M.J.__.PDF",
+    "precedential_statuses": "PUBLISHED",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "neutral_citations": null,
+    "west_citations": null,
+    "case_name_shorts": "OLSEN"
+  },
+  {
+    "case_dates": "2019-08-28",
+    "case_names": "IN RE A.H. PETITION - ORDER (79 M.J. 672)",
+    "download_urls": "https://media.defense.gov/2019/Oct/07/2002191513/-1/-1/0/IN%20RE%20A.H.%20PETITION%20-%20ORDER.PDF",
+    "precedential_statuses": "PUBLISHED",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "neutral_citations": "79 M.J. 672",
+    "west_citations": null,
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2019-07-03",
+    "case_names": "UNITED STATES v. GOODELL",
+    "download_urls": "https://media.defense.gov/2019/Jul/03/2002153483/-1/-1/0/UNITED%20STATES%20V.%20GOODELL%20_M.J._.PDF",
+    "precedential_statuses": "PUBLISHED",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "neutral_citations": null,
+    "west_citations": null,
+    "case_name_shorts": "GOODELL"
+  },
+  {
+    "case_dates": "2019-07-02",
+    "case_names": "United States V GUZMAN",
+    "download_urls": "https://media.defense.gov/2019/Jul/03/2002153369/-1/-1/0/U.S.%20V.%20GUZMAN%20(UNPUBLISHED).PDF",
+    "precedential_statuses": "PUBLISHED",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "neutral_citations": null,
+    "west_citations": null,
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2019-05-15",
+    "case_names": "UNITED STATES v. CORREA",
+    "download_urls": "https://media.defense.gov/2019/May/21/2002134458/-1/-1/0/UNITED%20STATES%20V.%20CORREA%20%20(MERITS).PDF",
+    "precedential_statuses": "PUBLISHED",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "neutral_citations": null,
+    "west_citations": null,
+    "case_name_shorts": "CORREA"
+  }
```

Comment:
It's a good practice to check the example compare json to make sure its generating the results you see in the HTML.